MidCamp/midcamp
# #3317911: Midcamp 2023

https://www.drupal.org/project/midcamp/issues/3317911

## Description

- Exports production config containing miscellaneous 2023 event prep items performed directly in Prod
- Updates configuration, custom code per [Preparing MidCamp.org for a new event year](https://github.com/MidCamp/midcamp/wiki/Preparing-MidCamp.org-for-a-new-event-year) documentation

## To Test

1. Visit the PR environment https://nginx.feature-midcamp2023.midcamp-org.us2.amazee.io/ an confirm the site looks as expected
2. Visit the organizer page at https://nginx.feature-midcamp2023.midcamp-org.us2.amazee.io/2023/organizers and confirm it is displaying Organizers assigned to the 2023 event year (missing images is expected in this environment).
3. Create content and do Midcamp things and confirm they behave as you expect.

## Deployment tasks

Additional setup will need to be performed once this code lands in Production.  These steps will include:

- [ ] Delete the Basic Page at `/2023/organizers`, which is overriding the View page
- [ ] Add the MidCamp 2023 taxonomy term to some user accounts (`field_organizer_year` and optionally `field_leadership_year`) with the Organizer role.  Navigate to `/2023/organizers` and observe they are now appearing in the display there
